core/interface/static/css/projectalice.css

```diff
@@ -1,714 +1,246 @@
-
-.pageTitle {
-	font-family: 'marskeregular', Arial, sans-serif;
-	margin-left: 15px;
-	font-size: 2em;
-	display: flex;
-	align-items: center;
-	height: 100%;
-	flex-grow: 1;
-}
-
-.updateChannelMarker {
-	font-family: 'marskeregular', Arial, sans-serif;
-	margin-left: 15px;
-	font-size: 1em;
-	display: flex;
-	align-items: center;
-	height: 100%;
-	font-style: italic;
-	color: red;
-	flex-grow: 1;
-	justify-content: flex-end;
-	margin-right: 15px;
-}
-
-.content {
-	background-color: #1e1e1e;
-	position: fixed;
-	top: 54px;
-	left: 154px;
-	right: 0;
-	bottom: 2px;
-	margin-right: 2px;
-}
-
-.toolbar {
-	width: 100%;
-	height: 40px;
-	text-align: right;
-	padding-top: 5px;
-	border-bottom: 15px;
-}
-
-.toolbarButton {
-	color: #d1d1d1;
-	font-size: 2em;
-	margin-right: 10px;
-}
-
-.toolbarButton:hover {
-	color: yellow;
-	cursor: pointer;
-}
-
-.modulesStore {
-	position: fixed;
-	top: 96px;
-	left: 154px;
-	right: 0;
-	bottom: 0;
-	margin: 5px;
-	overflow-y: auto;
-	overflow-x: hidden;
-	display: none;
-	flex-wrap: wrap;
-	align-content: flex-start;
-}
-
-.moduleStoreWait {
-	width: 100%;
-	color: #d1d1d1;
-	font-size: 10em;
-	text-align: center;
-	margin-top: 30px;
-}
-
-.moduleStoreModuleTile {
-	font-family: 'marskeregular', Arial, sans-serif;
-	display: flex;
-	flex-wrap: wrap;
-	align-items: flex-start;
-	background-color: #636363;
-	margin: 10px;
-	width: 300px;
-	height: 300px;
-	color: #d1d1d1;
-	align-content: flex-start;
-	box-shadow: 0 0 20px 5px #141414;
-}
-
-.modulesStoreModuleTitle {
-	background-color: #343434;
-	height: 30px;
-	width: 100%;
-	display: flex;
-	align-items: center;
-	padding-left: 5px;
-	font-size: 1.25em;
+	margin: 0;
+	-webkit-box-sizing: border-box;
+	-moz-box-sizing: border-box;
 	box-sizing: border-box;
 }
 
-.modulesStoreModuleAuthor,
-.modulesStoreModuleVersion,
-.modulesStoreModuleCategory {
-	width: 100%;
-	font-size: 1em;
-	margin: 5px;
-}
-
-.moduleStoreModuleDescription {
-	font-family: neogreyregular, sans-serif;
+html,
+body {
+	background-color: var(--bg-color);
 	width: 100%;
-	font-size: 0.75em;
-	margin: 5px;
-	background-color: #343434;
-	height: 110px;
-	min-height: 110px;
-	box-sizing: border-box;
-	padding: 5px;
-	overflow-x: hidden;
-	overflow-y: auto;
-}
-
-.moduleStoreModuleDownload {
-	height: 50px;
-	width: 50%;
-	margin: 10px auto auto auto;
-	background-color: #343434;
-	align-content: center;
-	align-items: center;
-	justify-items: center;
-	justify-content: center;
-	font-size: 2em;
-}
-
-.moduleStoreModuleDownload:hover {
-	cursor: pointer;
-	background-color: #636363;
-	color: yellow;
-}
-
-.moduleStoreModuleButtonAnimation {
-	display: none;
-	height: 50px;
-	width: 50%;
-	margin: 10px auto auto auto;
-	background-color: #343434;
-	align-content: center;
-	align-items: center;
-	justify-items: center;
-	justify-content: center;
-	font-size: 2em;
-}
-
-.moduleStoreModuleDownloadButton {
-	display: flex;
-}
-
-#closeModuleStore {
-	display: none;
+	min-height: 100% !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/app/ProjectAlice/ProjectAlice/pullRequest?prid=4618587)